model/src/main/java/org/projectnessie/model/HiveTable.java

```diff
@@ -33,4 +36,50 @@
 
   public abstract List<byte[]> getPartitions();
 
+  /**
+   * Because of the List of byte arrays Immutables doesn't generate a correct hashcode.
+   *
+   * <p> We hand compute the List elements to ensure the byte arrays hash codes are consistent.
+   */
+  @Override
+  public int hashCode() {
+    int h = 5381;
```

Comment:
I'd prefer the "standard" way common in the JDK. I.e. initialize with `1` and use `h = (31 * h) + xyz`, like `java.util.Arrays#hashCode(java.lang.Object[])` does.

Reply:
cool, fixed. That was a direct copy from immutables but I agree, we should stick to the standard